aws-google.gemspec

```diff
@@ -21,14 +22,14 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'aws-sdk-core', '~> 3.130'
-  spec.add_dependency 'google-apis-core'
-  spec.add_dependency 'launchy', '~> 2'
+  spec.add_dependency 'aws-sdk-core', '~> 3.209.1'
```

Comment:
What happens if this doesn't match the exact version in code-dot-org/Gemfile? Relevant to the Ruby 3.3 pre-upgrade etc. If at all possible, I'd love to land the Ruby 3.3 pre-upgrade before merging this PR, because it feels like they might have some interaction (the pre-upgrade PR also changes aws-sdk-core version, to address the rather crazy poste issues Elijah & I were seeing)

Reply:
If code-dot-org uses a newer version, will bundle just install both?

Reply:
My understanding, which could easily be wrong, is that you only get one version of each gem in your ruby projects. Bundle can install lots of versions side by side, but when we load `dashboard-server`, there will be only one version in memory, and it'll have to be shared between aws-google and code-dot-org.

My impression has been that libraries in Ruby often use > matches rater than ~> in order to not overly constrain dep solving for apps that use them.

Reply:
I just checked `code-dot-org/Gemfile.lock`, and it looks like its actually `aws-google` that constraints our exact aws-sdk-core Gemfile version, so changing this seems safe. That said, I notice AWS' own `aws-sdk-*` deps don't pin exact versions of aws-sdk-core, maybe worth copying their format to avoid this issue in the future?

aws-google pins relatively tightly:
```
aws-google (0.2.0)
      aws-sdk-core (~> 3.130)
```

Whereas amazon's own deps only pin to the major version, and require that aws-sdk-core be at least as recent as them:
```
aws-sdk-cloudfront (1.55.0)
      aws-sdk-core (~> 3, >= 3.119.0)
```

<img width="347" alt="image" src="https://github.com/user-attachments/assets/61a050a3-f6db-4aab-ab62-376192bdcfb3">

Reply:
TL;DR: this should work totally fine, but maybe pinning even less would be better for the next time